### ISSUE TITLE
Is anyone able to reproduce the results from the paper?

### ISSUE THREAD

**Issue by DevDynamo-web** [NaVi-start/Sign-IDD#5]
I strictly followed the configuration provided by the authors, but the training process stopped prematurely after only 805 epochs, and the best DEV DTW score achieved was just 11.45. The other test metrics are shown below:

<img width="1350" height="353" alt="Image" src="https://github.com/user-attachments/assets/750b349e-ec5c-4351-b773-da1318df2392" />

These results differ significantly from those reported in the paper, and I’m unsure where the issue might lie. Would it be possible for the authors to share the processed input data, pretrained model weights, or the train.log file to help verify the setup? If anyone has successfully reproduced the reported results, I would greatly appreciate any guidance or suggestions. Thank you in advance!

**Comment by NaVi-start** [comment]
Steps: 2000 Loss: 0.49677| DTW: 14.934| LR: 0.001000 *
Steps: 4000 Loss: 0.40286| DTW: 14.669| LR: 0.001000 *
Steps: 6000 Loss: 0.36094| DTW: 13.474| LR: 0.001000 *
Steps: 8000 Loss: 0.35264| DTW: 12.838| LR: 0.001000 *
Steps: 10000 Loss: 0.30812| DTW: 12.063| LR: 0.001000 *
Steps: 12000 Loss: 0.32229| DTW: 12.370| LR: 0.001000 
Steps: 14000 Loss: 0.30283| DTW: 11.723| LR: 0.001000 *
Steps: 16000 Loss: 0.31739| DTW: 11.572| LR: 0.001000 *
Steps: 18000 Loss: 0.30129| DTW: 11.553| LR: 0.001000 *
Steps: 20000 Loss: 0.30414| DTW: 11.471| LR: 0.001000 *
Steps: 22000 Loss: 0.30037| DTW: 11.610| LR: 0.001000 
Steps: 24000 Loss: 0.29704| DTW: 11.490| LR: 0.001000 
Steps: 26000 Loss: 0.31118| DTW: 11.608| LR: 0.001000 
Steps: 28000 Loss: 0.30783| DTW: 11.539| LR: 0.001000 
Steps: 30000 Loss: 0.31509| DTW: 11.528| LR: 0.001000 
Steps: 32000 Loss: 0.31157| DTW: 11.397| LR: 0.001000 *
Steps: 34000 Loss: 0.31730| DTW: 11.455| LR: 0.001000 
Steps: 36000 Loss: 0.31707| DTW: 11.325| LR: 0.001000 *
Steps: 38000 Loss: 0.32206| DTW: 11.492| LR: 0.001000 
Steps: 40000 Loss: 0.33370| DTW: 11.453| LR: 0.001000 
Steps: 42000 Loss: 0.34612| DTW: 11.459| LR: 0.001000 
Steps: 44000 Loss: 0.33458| DTW: 11.391| LR: 0.001000 
Steps: 46000 Loss: 0.34771| DTW: 11.331| LR: 0.001000 
Steps: 48000 Loss: 0.34383| DTW: 11.478| LR: 0.001000 
Steps: 50000 Loss: 0.33230| DTW: 11.346| LR: 0.001000 
Steps: 52000 Loss: 0.34470| DTW: 11.312| LR: 0.001000 *
Steps: 54000 Loss: 0.35600| DTW: 11.484| LR: 0.001000 
Steps: 56000 Loss: 0.36576| DTW: 11.374| LR: 0.001000 
Steps: 58000 Loss: 0.34529| DTW: 11.448| LR: 0.001000 
Steps: 60000 Loss: 0.34794| DTW: 11.417| LR: 0.001000 
Steps: 62000 Loss: 0.36944| DTW: 11.417| LR: 0.001000 
Steps: 64000 Loss: 0.35142| DTW: 11.347| LR: 0.001000 
Steps: 66000 Loss: 0.36269| DTW: 11.438| LR: 0.001000 
Steps: 68000 Loss: 0.35518| DTW: 11.441| LR: 0.001000 
Steps: 70000 Loss: 0.37261| DTW: 11.381| LR: 0.000700 
Steps: 72000 Loss: 0.36838| DTW: 11.348| LR: 0.000700 
Steps: 74000 Loss: 0.37633| DTW: 11.411| LR: 0.000700 
Steps: 76000 Loss: 0.36828| DTW: 11.444| LR: 0.000700 
Steps: 78000 Loss: 0.37437| DTW: 11.387| LR: 0.000700 
Steps: 80000 Loss: 0.37517| DTW: 11.398| LR: 0.000700 
Steps: 82000 Loss: 0.37784| DTW: 11.297| LR: 0.000700 *
Steps: 84000 Loss: 0.37645| DTW: 11.294| LR: 0.000700 *
Steps: 86000 Loss: 0.36947| DTW: 11.297| LR: 0.000700 
Steps: 88000 Loss: 0.37703| DTW: 11.352| LR: 0.000700 
Steps: 90000 Loss: 0.37238| DTW: 11.274| LR: 0.000700 *
Steps: 92000 Loss: 0.37642| DTW: 11.337| LR: 0.000700 
Steps: 94000 Loss: 0.38298| DTW: 11.286| LR: 0.000700 
Steps: 96000 Loss: 0.39151| DTW: 11.411| LR: 0.000700 
Steps: 98000 Loss: 0.38289| DTW: 11.409| LR: 0.000700 
Steps: 100000 Loss: 0.38490| DTW: 11.335| LR: 0.000700 
Steps: 102000 Loss: 0.38258| DTW: 11.352| LR: 0.000700 
Steps: 104000 Loss: 0.37136| DTW: 11.356| LR: 0.000700 
Steps: 106000 Loss: 0.38420| DTW: 11.291| LR: 0.000490 
Steps: 108000 Loss: 0.38971| DTW: 11.344| LR: 0.000490 
Steps: 110000 Loss: 0.38072| DTW: 11.274| LR: 0.000490 *
Steps: 112000 Loss: 0.39898| DTW: 11.300| LR: 0.000490 
Steps: 114000 Loss: 0.38278| DTW: 11.348| LR: 0.000490 
Steps: 116000 Loss: 0.37524| DTW: 11.368| LR: 0.000490 
Steps: 118000 Loss: 0.40831| DTW: 11.416| LR: 0.000490 
Steps: 120000 Loss: 0.39690| DTW: 11.402| LR: 0.000490 
Steps: 122000 Loss: 0.40244| DTW: 11.369| LR: 0.000490 
Steps: 124000 Loss: 0.39794| DTW: 11.337| LR: 0.000490 
Steps: 126000 Loss: 0.39160| DTW: 11.393| LR: 0.000343 
Steps: 128000 Loss: 0.39226| DTW: 11.336| LR: 0.000343 
Steps: 130000 Loss: 0.40177| DTW: 11.335| LR: 0.000343 
Steps: 132000 Loss: 0.38445| DTW: 11.331| LR: 0.000343 
Steps: 134000 Loss: 0.40297| DTW: 11.389| LR: 0.000343 
Steps: 136000 Loss: 0.40331| DTW: 11.369| LR: 0.000343 
Steps: 138000 Loss: 0.39116| DTW: 11.351| LR: 0.000343 
Steps: 140000 Loss: 0.37892| DTW: 11.367| LR: 0.000343 
Steps: 142000 Loss: 0.38319| DTW: 11.326| LR: 0.000240 
Steps: 144000 Loss: 0.38581| DTW: 11.349| LR: 0.000240 
Steps: 146000 Loss: 0.39981| DTW: 11.329| LR: 0.000240 
Steps: 148000 Loss: 0.40546| DTW: 11.351| LR: 0.000240 
Steps: 150000 Loss: 0.40182| DTW: 11.332| LR: 0.000240 
Steps: 152000 Loss: 0.40226| DTW: 11.397| LR: 0.000240 
Steps: 154000 Loss: 0.39018| DTW: 11.367| LR: 0.000240 
Steps: 156000 Loss: 0.41002| DTW: 11.358| LR: 0.000240 
Steps: 158000 Loss: 0.40114| DTW: 11.349| LR: 0.000168

**Comment by NaVi-start** [comment]
We have uploaded the validations file to the Configs folder. Different environment configurations may slightly affect the final result.

**Comment by DevDynamo-web** [comment]
<img width="1582" height="558" alt="Image" src="https://github.com/user-attachments/assets/c1024f88-b3e4-473e-b39a-7a897ed6045a" />The validation code you provided may have issues. I used the ground truth on the dev and test sets to evaluate FID, MPJPE, and MPJAE. They should all be zero, but I got the following values. Other metrics also do not exactly match the ground truth reported in the paper.

**Comment by ahnxoniii** [comment]
Hello,

I hope you’re doing well.
I was wondering if it would be possible to access or receive the training dataset used in your work.

I would really appreciate it if you could share it or let me know how I can obtain it.

Thank you very much!
